plugins/eduid_action.mfa/component.js

```diff
@@ -4,7 +4,6 @@ import { connect } from "react-redux";
 
 import i18n from "i18n-messages";
 import {
-  appFetching,
```

Comment:
I just want to double check that it's ok to go beyond commenting this out here and fully remove it

Reply:
Yes all are consciously removed. In particular: The appFetching action was not being used anywhere anymore